Packages/eu.netherlands3d.indicators/Runtime/Scripts/DossierVisualiser.cs

```diff
@@ -171,10 +183,77 @@ private PolygonVisualisation VisualizePolygon(string projectAreaId, int polygonI
         {
             var visualisationFromPolygon = CreateVisualisationFromPolygon(polygon);
             visualisationFromPolygon.gameObject.name = $"{projectAreaId}::{polygonIndex}";
-
+            visualisationFromPolygon.gameObject.AddComponent<DossierVisualisationClickHandler>().SetVisualiser(this);            
+            
             return visualisationFromPolygon;
         }
 
+        public void MoveSamplePointer(Vector3 worldPosition)
+        {
+            if(dossier.SelectedDataLayer != null && dossier.SelectedDataLayer.Value.frames != null)
```

Comment:
Style thingie: Inverting the if and using an early return will remove the need for an else and reduce complexity

Reply:
Fixed